Doc/library/math.rst

```diff
@@ -136,6 +136,15 @@ Number-theoretic and representation functions
    .. versionadded:: 3.5
 
 
+.. function:: lcm(a, b)
+
+   Return the least common multiple of integers *a* and *b*.  The value of
+   ``lcm(a, b)`` is the smallest non negative integer that is a multiple of
```

Comment:
Nitpick: `nonnegative` or `non-negative` rather than `non negative`, please. (I personally prefer the former, but it's a matter of taste, and both forms are already in use in the math module.)